gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/Constants.java

```diff
@@ -15,7 +15,7 @@ public abstract class Constants {
     
     public static final String WEB_TEMPLATE_DISTRIBUTION_VERSION = "2.5.1";

```

Comment:
Only reason I noticed is because I also had originally forgotten for my version 3... this should be 2.6.0 as well :)

```suggestion
    public static final String WEB_TEMPLATE_DISTRIBUTION_VERSION = "2.6.0";
```